<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrownWidget</class>
 <widget class="QWidget" name="CrownWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>271</width>
    <height>138</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>271</width>
    <height>138</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CrownWidget</string>
  </property>
  <widget class="VitaWidget" name="vitaWidget" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>65</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="materialLabel">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Материал:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>40</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="materialEdit">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>10</y>
     <width>150</width>
     <height>19</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>40</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Безпрагово изпиляване</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Циментиране в/у имплант</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Прагово изпиляване</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Завинтване в/у имплант</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VitaWidget</class>
   <extends>QWidget</extends>
   <header>VitaWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
